### DISCUSSION TITLE
It looks like you are trying to use Tree Data with Pagination but unfortunately that is simply not supported because of its complexity.

### DISCUSSION THREAD

**Discussion by Hariprakash-Jawahar** [ghiscoding/Angular-Slickgrid#1068] [Q&A] [answered]
HI im update slick grid 2.24.1 to 2.25.1 after that im getting expection like ( It looks like you are trying to use Tree Data with Pagination but unfortunately that is simply not supported because of its complexity. ) let me know which release  we can get pagination for tree data .

Reason for updating slickgrid--
angular slickgrid 2.24.1 having dependency of moment: 2.24.0 it having some vulnarabilties like (CVE-2022-24785  and CVE-2022-31129) there is any way to update moment in slick grid dependencies

**Comment by ghiscoding** [comment, marked as answer]
well the message should be easy enough to understand, you cannot use Pagination with Tree Data, if it's something else then sorry but I don't support old version (and this is very old)

**Reply by ghiscoding** [reply]
To update to a specific version of Moment, you can try [npm overrides](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides)

**Reply by Hariprakash-Jawahar** [reply]
Thanks it works!!

**Reply by ghiscoding** [reply]
also see issue #658 which described when and why I've added the new console error to advise users that Tree Data with Pagination is not supported and that error was introduced in 2.25.1, but regardless of when the error was added, both features were never supported together (and probably never will because of the complexity).
